builtin/credential/cert/path_certs.go

```diff
@@ -47,7 +47,22 @@ Must be x509 PEM encoded.`,
 					EditType: "file",
 				},
 			},
-
+			"ocsp_enabled": {
```

Comment:
Might be nice to allow operators to select if they want a FailClose or FailOpen behavior as well?